src/libraries/periphery/MorphoBalancesLib.sol

```diff
@@ -10,7 +10,7 @@ import {MorphoLib} from "./MorphoLib.sol";
 import {SharesMathLib} from "../SharesMathLib.sol";
 import {MarketParamsLib} from "../MarketParamsLib.sol";
 
-interface IMorphoMarketStruct {
+interface IMorphoMarketGetter {
```

Comment:
```suggestion
interface IMorpho {
```

Reply:
I don't think it's good for integrators as it would create a clash if they to use both interface...

Reply:
they could rename it, but as you want

Reply:
> note that this
> 
> ```solidity
> Market memory market = IMorphoMarketGetter(address(morpho)).market(id);
> ```
> 
> becomes
> 
> ```solidity
> Market memory market = morpho.market(id);
> ```

What you suggest forces the integrator to use the `IMorpho` defined locally instead of the `IMorpho` from `IMorpho.sol`. I am strongly against: what's useful with these libraries is that they are plug-and-play with the standard `IMorpho` interface

Reply:
Nice catch